packages/lesswrong/lib/collections/moderatorActions/helpers.ts

```diff
@@ -106,12 +106,19 @@ export function getReasonForReview(user: DbUser | SunshineUsersList, override?:
   if (override) return 'override';
 
   const fullyReviewed = user.reviewedByUserId && !user.snoozedUntilContentCount;
-  const neverReviewed = !user.reviewedByUserId && !user.reviewedAt;
+  /**
+   * This covers several cases
+   * 1) never reviewed users
+   * 2) users who were removed from the review queue and weren't previously reviewed
+   * 3) users who were removed from the review queue and *were* previously reviewed
+   * 1 & 2 look indistinguishable, 3 will have a non-null reviewedAt date
+   */ 
+  const unreviewed = !user.reviewedByUserId;
```

Comment:
This code makes me so so sad. See also: https://github.com/ForumMagnum/ForumMagnum/issues/6964